installed-versions.json

```diff
@@ -38,8 +38,8 @@
       "resolved": "https://registry.npmjs.org/sass-loader/-/sass-loader-10.2.0.tgz"
     },
     "sass": {
-      "version": "1.44.0",
-      "resolved": "https://registry.npmjs.org/sass/-/sass-1.44.0.tgz"
+      "version": "1.45.0",

```

Comment:
Not sure if Vuetify triggers the sass `/ is deprecated, use math.division()` warnings or not but in material and bootstrap it is unsufferable.